core/Cargo.toml

```diff
@@ -45,6 +45,7 @@ hmdb = "0.1.12"
 lazy_static = "1.4.0"
 lockbook-models = { path = "libs/models" }
 lockbook-crypto = { path = "libs/crypto" }
+sublime_fuzzy = "0.7.0"
```

Comment:
I can be sold that this is better, should we just use this for paths too?

Reply:
Not sure, should I change it in this PR or leave it to another one?